lms/views/status.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+
+import logging
+
+from pyramid.httpexceptions import HTTPInternalServerError
+from pyramid.view import view_config
+
+
+log = logging.getLogger(__name__)
+
+
+@view_config(route_name='status', renderer='json')
+def status(request):
+    try:
+        request.db.execute('SELECT 1')
+        return {'status': 'okay'}
+    except Exception:  # pylint:disable=broad-except
+        log.exception("Executing a simple database query failed:")
+        raise HTTPInternalServerError('Database connection failed')
```

Comment:
This returns a non-JSON 500, which is the same as what bouncer does. h actually returns a JSON object representing failure and 500s, I haven't bothered to implement that here. [The old lti app actually returned failure-representing JSON in a 200 response on failure](https://github.com/hypothesis/lti/blob/master/lti/views/status.py) but I don't see how the load balancer was configured to parse that JSON, rather than just believing the HTTP status code. Maybe AWS supports this by default? Or maybe it was wrong.

Reply:
Assuming the health check endpoint works the same way as ELB's health checking does, only the status code is considered: https://hyp.is/dHpS5OAkEeeT3efwt7HIfg/docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-healthchecks.html